gtk-sys/src/lib.rs

```diff
@@ -14222,14 +14221,8 @@ pub struct GtkStyle {
     pub font_desc: *mut pango::PangoFontDescription,
     pub xthickness: c_int,
     pub ythickness: c_int,
-    pub background: *mut [*mut cairo::cairo_pattern_t; 5],
-    pub attach_count: c_int,
-    pub visual: *mut gdk::GdkVisual,
-    pub private_font_desc: *mut pango::PangoFontDescription,
-    pub rc_style: *mut GtkRcStyle,
-    pub styles: *mut glib::GSList,
-    pub property_cache: *mut glib::GArray,
-    pub icon_factories: *mut glib::GSList,
+    _truncated_record_marker: c_void,
```

Comment:
`background` should be there no?

Reply:
It really incomplete 
`#[repr(C)] pub struct cairo_pattern_t(c_void);`